ci/linux/Dockerfile

```diff
@@ -76,5 +78,5 @@ RUN cd /tmp && \
   cd /tmp/SDL_sound  && \
   mkdir /tmp/SDL_sound/build && \
   cd /tmp/SDL_sound/build && \
-  cmake ..  && make && make install && \
+  cmake -DSDL2_DIR=/usr/local/lib/cmake/SDL2 ..  && make -j4 && make install && \
```

Comment:
why does the path to SDL2 need to be hardcoded here?

Reply:
SDL2_DIR has to be specified if SDL2 is configured using CMake generated makefile instead of ./configure, before installation. The CMake build support all flags relative to reducing direct linking libraries in SDL2, so if I change this in the future things won't fail mysteriously. This also specifies which SDL2 I want to be linked with SDL2_sound in an obvious way, which is not the system one, it's the one I built.